src/main.zig

```diff
@@ -1262,21 +1262,21 @@ fn buildOutputType(
                         want_tsan = true;
                     } else if (mem.eql(u8, arg, "-fno-sanitize-thread")) {
                         want_tsan = false;
-                    } else if (mem.eql(u8, arg, "-fLLVM")) {
+                    } else if (mem.eql(u8, arg, "-fllvm")) {
```

Comment:
did you check if this breaks homebrew

Reply:
what does this have to do with homebrew?

Reply:
Ah i thought it'd use this flag but it does not 

Evidently it does use llvm 15 though 

https://github.com/Homebrew/homebrew-core/blob/master/Formula/zig.rb#L25-L40